docs/development/issue_labeling.md

```diff
@@ -0,0 +1,228 @@
+# Issue labeling
+
+We try to keep issues well-classified by labeling them appropriately.
+Only members/collaborators with the required rights can apply labels.
+
+## Basic knowledge about Renovate
+
+You should know about platforms, package managers, datasources and versioning to label issues effectively.
+
+- To learn about platforms, read the [Renovate docs on Platforms](https://docs.renovatebot.com/modules/platform/).
+- To learn about managers, read the [Renovate docs on Managers](https://docs.renovatebot.com/modules/manager/).
+- To learn about datasources, read the [Renovate docs on Datasources](https://docs.renovatebot.com/modules/datasource/).
+- To learn more about versioning, read the [Renovate docs on Versioning](https://docs.renovatebot.com/modules/versioning/).
+
+## Label categories
+
+### Severity
+
+<details>
+    <summary>Severity</summary>
+
+    breaking
+    major
+
+</details>
+
+QUESTION: When/who should use these severity labels?
```

Comment:
Breaking is added if we determine it's a backwards-incompatible change that requires us to publish a new major release.